sklearn/utils/estimator_checks.py

```diff
@@ -643,6 +644,48 @@ def check_fit2d_predict1d(name, estimator_orig):
                                  getattr(estimator, method), X[0])
 
 
+@ignore_warnings(category=(DeprecationWarning, FutureWarning))
+def check_methods_subset_invariance(name, estimator_orig):
+    # check that method gives invariant results if applied
+    # one by one or on all elements together.
```

Comment:
I would slightly rephrase the comment by replacing one by one by something like "if applied on mini bathes or the whole set"